app/workers/suspend_inactive_accounts_worker.rb

```diff
@@ -1,9 +1,13 @@
 # frozen_string_literal: true
 
 class SuspendInactiveAccountsWorker
-  include Sidekiq::Job
+  include Sidekiq::IterableJob
 
-  def perform
-    AutoAccountDeletionQueries.should_be_suspended.find_each(&:suspend!)
+  def build_enumerator(*_args, cursor:)
+    active_record_records_enumerator(AutoAccountDeletionQueries.should_be_suspended, cursor:)
+  end
+
+  def each_iteration(item, *_args)
+    item.suspend
```

Comment:
Similarly, `*_args` seems to account for any parameters that `#build_enumerator` accepts. But since we have none there, I think this should probably work
```suggestion
  def each_iteration(account)
    account.suspend
```

Reply:
https://github.com/3scale/porta/pull/4114/commits/414e63d9d37d6d4e88175c0babd25b6583870511